<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListOptions</class>
 <widget class="QDialog" name="ListOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="image">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="displayButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Display list</string>
   </property>
  </widget>
  <widget class="QPushButton" name="learningModeBButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Learning mode B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="learningModeAButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Learning mode A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>530</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addNewWordButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Add new word</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goBackButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
